doc/platform/app/app_roles.rst

```diff
@@ -238,7 +215,60 @@ This means that all the dependencies of a role should be defined in the ``roles`
 
 You can find the full example here: `application_role_cfg <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/config/instances.enabled/application_role_cfg>`_.
 
+.. _roles_create_custom_role_on_event_callback:
+
+On_event callback
+~~~~~~~~~~~~~~~~~
+
+Since version :doc:`3.4.0 </release/3.4.0>`, you can define the ``on_event`` callback for custom roles. The ``on_event`` callback is called
+every time a ``box.status`` system event is broadcasted.
+If multiple custom roles have the ``on_event`` callback defined, these callbacks are called one after another in the order
+defined by roles dependencies.
+
+The ``on_event`` callback returns 3 arguments, when it is called:
+
+- ``config``, which contains the configuration of the role;
+
+- ``key``, which reflects the trigger event and is set to:
```

Comment:
`key` and `value` are too general, they don't give much information for a reader. How about `event_name` and `event_data`?